tests/externallib_test.php

```diff
@@ -18,6 +18,8 @@
 
 global $CFG;
 
+use local_raise_external;
```

Comment:
I would avoid writing this test case to use interfaces from our other plugin. Instead just limit the assumptions at the database level that `local_raise_user` exists, and create dummy rows in the table directly.